reference/conanfile/attributes.rst

```diff
@@ -1670,3 +1670,24 @@ When ``True`` it enables the new run in a subsystem bash in Windows mechanism. :
     class FooRecipe(ConanFile):
         ...
         win_bash = True
+
+.. _test_type:
+
+test_type
+---------
+
+.. warning::
+
+    Components is a **experimental** feature subject to breaking changes in future releases.
+
+Available since: `1.44.0 <https://github.com/conan-io/conan/releases/tag/1.44.0>`_
+
+This attribute allows testing requirements and build requiments explicitly on test package.
```

Comment:
I think it should be explicitly stated that it might be a list (comma separated), e.g. the following should be a valid example:
```
test_type = 'requires', 'build_requires', `explicit`
``

Reply:
No, only ``explicit`` should be documented from now on.